wikidata/datavalue.py

```diff
@@ -195,6 +196,8 @@ def time(self,
             precision = value['precision']
         except KeyError:
             raise DatavalueError('precision field is missing', datavalue)
+        if precision == 9:
```

Comment:
A comment would be helpful.

```suggestion
        if precision == 9:  # year only
```